lib/v8.js

```diff
@@ -14,6 +14,10 @@
 
 'use strict';
 
+if (process.jsEngine === 'chakracore') {
+    return;
```

Comment:
Looks like you have four spaces here, this is probably the cause of the linter failure in CI.